.github/workflows/update-data.yml

```diff
@@ -7,6 +7,21 @@ on:
       - "main"
     tags: [ 'v*' ]
   workflow_dispatch:
+    inputs:
+      target:
+        description: 'dev or prod'
+        required: true
+        default: 'dev'
+        type: choice
+        options: [dev, prod]
+      branch_ref:
+        description: 'Branch to use for dev (ignored for prod)'
+        required: false
+        default: 'main'
```

Comment:
I think this argument is probably not necessary. When you run workflows there's always the `Use workflow from` option that allows you to select a branch or tag, and then I believe you can access that with `${{ github.ref_name }}`. I've left a suggestion here, and one below that I think should allow us to remove this.

```suggestion
```